apps/aetestframework/.formatter.exs

```diff
@@ -0,0 +1,4 @@
+# Used by "mix format"
```

Comment:
i assume this file is not needed and these rules should be included in the parent formatter config